src/price_data.py

```diff
@@ -26,6 +26,8 @@
 import config
 import misc
 from transaction import *
+from core import kraken_pair_map
+import bisect
```

Comment:
I like to order my imports in three categories which are ordered each alphabetically.

1. standardlib
2. third-party
3. internal

This would look like
```
import bisect

import config
from core import kraken_pair_map
import misc
from transaction import *
```

This helps to get a quick overlook over the imports in the file.
Just my taste, you do not have to bend for this ;)

Reply:
Got it!